scripts/mixpanel_traffic.py

```
@@ -0,0 +1,189 @@
+        data=b"grant_type=client_credentials",
+        headers={
+            "Authorization": f"Basic {creds}",
+            "Content-Type": "application/x-www-form-urlencoded",
+        },
+    )
+    with urllib.request.urlopen(req, timeout=10) as r:
+        return json.loads(r.read())["access_token"]
+
+
+def call(token: str, path: str) -> int:
+    req = urllib.request.Request(f"{BASE}{path}", headers={"Authorization": f"Bearer {token}"})
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)